README.md

```diff
@@ -26,7 +26,7 @@ func updateCallback(rev string) {
 func main() {
     // Initialize the watcher.
     // Use the endpoint of etcd cluster as parameter.
-    w, _ := etcdwatcher.NewWatcher("http://127.0.0.1:2379")
+    w, _ := etcdwatcher.NewWatcher([]string{"http://127.0.0.1:2379"}, "keyname")
```

Comment:
What's `"keyname"`?

Reply:
A placeholder for the key to watch on Etcd